.circleci/config.yml

```diff
@@ -132,6 +132,223 @@ commands:
       - run: echo 'export PATH="/home/circleci/.local/bin:$PATH"' >> "$BASH_ENV"
 
 jobs:
+  build_cli:
```

Comment:
Is this purely a mechanical change? Just moving this section of code up?

Reply:
just grouping more logically (imo)